src/v2/guide/components-registration.md

```diff
@@ -150,13 +150,13 @@ export default {
 }
 ```
 
-Now both `ComponentA` and `ComponentC` can be used inside `ComponentB`'s template.
+Ahora ambos `ComponentA` y `ComponentC` pueden ser usados dentro de la plantilla del `ComponentB`.
 
-### Automatic Global Registration of Base Components
+### Registro global automático de los componentes básicos
 
-Many of your components will be relatively generic, possibly only wrapping an element like an input or a button. We sometimes refer to these as [base components](../style-guide/#Base-component-names-strongly-recommended) and they tend to be used very frequently across your components.
+Muchos de sus componentes serán relativamente genéricos, posiblemente sólo envolviendo un elemento como un input o un botón. A veces nos referimos a ellos como [componentes base](../style-guide/#Base-component-names-strongly-recommended) y tienden a ser usados muy frecuentemente a través de sus componentes.
```

Comment:
I'm with the accent here in `sólo` as it refers to `solamente`, and that one should have an accent

Reply:
Then you should remove the accent, right?